cfep-12.md

```diff
@@ -0,0 +1,33 @@
+
+<table>
+<tr><td> Title </td><td> Removing packages that violate the terms of the source package</td>
+<tr><td> Status </td><td> Draft </td></tr>
+<tr><td> Author(s) </td><td> Isuru Fernando &lt;isuruf@gmail.com&gt;</td></tr>
+<tr><td> Created </td><td> Feb 26, 2020</td></tr>
+<tr><td> Updated </td><td> Feb 26, 2020</td></tr>
+<tr><td> Discussion </td><td> NA </td></tr>
+<tr><td> Implementation </td><td> NA </td></tr>
+</table>
+
+## Abstract
+
+Source packages that we use to create conda packages have terms and conditions that allow
+us to redistribute the package in binary form. This CFEP codifies the conduct when these
+violations are reported.
+
+## Implementation
+
+When a violation is brought to the attention of the Core team, the feedstock maintenance
+team should be made aware by the Core team in a github issue on the feedstock and notified using the
+github handles of the maintenance team members.
+
+A week would be given to the maintenance team to respond to the claim and
+a week would be given to address the issue.
```

Comment:
Should this say "and another week would be given to address the issue"?

Reply:
Done. Also added a sentence saying that the team may ask for it to be extended.